### ISSUE TITLE
Demo not working

### ISSUE THREAD

**Issue by gsprs** [VPenkov/okayNav#41]
Hi,

I download the zip from GitHub and open in a browser the file dist/index.html and it does not show the header menu correctly. Items are one above the other and thus I cannot even make the demo work.

Can you please fix it ?
Thank you

**Comment by gsprs** [comment]
By the way I saw that the cdn to jquery was not working in the demo so I replaced it with a valid jquery cdn but it is still not working.
Tested with several browsers.

**Comment by DJTB** [comment]
The minified css provided with the demo doesn't actually contain all the necessary styles.

As a temporary fix - you could recompile the less, or reference the unminified css in the head of `dist/index.html`

```
    <link rel="stylesheet" href="css/common.min.css">
    <link rel="stylesheet" href="../app/css/okayNav-base.css">
    <link rel="stylesheet" href="../app/css/okayNav-theme.css">
```

**Comment by luizbills** [comment]
workings perfect here:
https://rawgit.com/VPenkov/okayNav/master/dist/index.html

**Comment by DJTB** [comment]
@luizbills I submitted a PR and it's now been merged - so this issue is outdated.

@VPenkov do you want close this - fixed by 910c22e
